Dockerfile.debian_package

```diff
@@ -0,0 +1,22 @@
+FROM debian:9
+
+# https://dotnet.microsoft.com/download/linux-package-manager/debian9/sdk-current
+RUN apt-get update \
+ && apt-get install -y apt-transport-https debhelper gnupg wget \
+ && wget -qO- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.asc.gpg \
+ && mv microsoft.asc.gpg /etc/apt/trusted.gpg.d/ \
+ && wget -q https://packages.microsoft.com/config/debian/9/prod.list \
```

Comment:
cannot you do it in one line like `wget -q http://url -O /etc/apt/..../microsoft-prod.list` ?.. why shuffle in two steps?

Reply:
I personally like splitting it like this if only to make it easier to read, but no big deal.

Reply:
I copied/pasted from the commented link as per Micrsoft's instructions.